lib/reporters/dots.js

```diff
@@ -32,15 +32,9 @@ function DotsReporter (formatError, reportSlow, useColors, browserConsoleLogOpti
     this.writeCommonMsg(this.renderBrowser(browser) + '\n')
   }
 
-  this.onRunComplete = function (browsers, results) {
-    if (browsers.length > 1 && !results.disconnected && !results.error) {
-      if (!results.failed) {
-        this.write(this.TOTAL_SUCCESS, results.success)
-      } else {
-        this.write(this.TOTAL_FAILED, results.failed, results.success)
-      }
-    }
-  }
+  this.onRunComplete = function (browsers, results) {}
```

Comment:
Overriding the base class here should cause the dots reporter to no longer report success or failure correct?

Reply:
Because the base will always be the default, so the failure or success will still be handled and printed to the console. 
Also from the results of using these reporters together,  just get duplicate messages are printed several times (which you can see from the referenced PR in angular-cli), is it still follows the design?

Reply:
> Also from the results of using these reporters together, just get duplicate messages are > printed several times (which you can see from the referenced PR in angular-cli), does it still follow the design?

Yes, two reporters should give two reports.  If you want one report, specify one reporter.

Reply:
thanks @johnjbarton , but I think if it's for specific outputs like progress in the progress reporter, that's reasonable, but for the total message or the error message which are same, just a little redundant.

Reply:
thanks @johnjbarton